### ISSUE TITLE
games run very fast on web

### ISSUE THREAD

**Issue by kavika13** [raysan5/raylib.com#17]
You have a fix for this in this commit: https://github.com/raysan5/raylib-games/commit/553c7486c18024cec5befcdb6d570c92815d5700

But I think the samples in use on the website haven't been rebuilt since then?

**Comment by raysan5** [comment]
@kavika13 Yeah, sorry, I haven't rebuilt them yet, I was waiting for raylib 3.5 release.
